Unity/Assets/NativeScript/Bindings.cs

```diff
@@ -49,19 +44,13 @@ public static void Init(int maxObjects)
 				objects = new object[maxObjects + 1];
 
```

Comment:
Oops it looks like I forgot to add the code that initializes the three variables I added above.